crates/node/src/lib.rs

```diff
@@ -438,14 +438,12 @@ impl Node {
                 application_id: context.application_id,
             });
         };
-        for (proposal_id, actions) in &outcome.proposals {
-            // todo deserialize actions into Vec<ProposalAction>
-            let action = ProposalAction::Transfer {
-                receiver_id: "vuki.testnet".into(),
-                amount: 0,
-            };
-            let actions = vec![action];
 
+        for (proposal_id, actions) in &outcome.proposals {
+            let actions: Vec<ProposalAction> = from_slice(&actions).map_err(|e| {
+                error!(%e, "Failed to deserialize proposal actions.");
+                CallError::InternalError
+            })?;
             drop(
                 self.ctx_manager
```

Comment:
why are we dropping? I suspect that's a result? it should be handled

same as below

Reply:
updated